action-server/db/base.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+TRACKER_STORE_USER = os.environ.get("SQL_TRACKER_STORE_USER")
+TRACKER_STORE_PASSWORD = os.environ.get("SQL_TRACKER_STORE_PASSWORD")
+TRACKER_STORE_URL = os.environ.get("SQL_TRACKER_STORE_URL")
+TRACKER_STORE_PORT = os.environ.get("SQL_TRACKER_STORE_PORT", "5432")
+TRACKER_STORE_DB = os.environ.get("SQL_TRACKER_STORE_DB")
+
+CONNECTION_POOL_SIZE = int(os.getenv("SQLALCHEMY_CONNECTION_POOL_SIZE", 15))
```

Comment:
Why use `os.environ.get` above and `os.getenv` here?

Reply:
i didn't even realized. will uniformize.